js/views/appView.js

```diff
@@ -23,6 +23,9 @@ define([
       $("body").on("rightAnswer",function(event,data){
         _this.countDown.clear().run()
       })
+	  $("body").on("wrongAnswer",function(event,data){
```

Comment:
This could be more specific and included in the events section of the view so that you don't have to 'dirty' up your body and worry about things bubbling up.